### PR TITLE
Original changes

### PR DESCRIPTION
**Pull request by ZEDz318**
Changes that were added when initially loading repository

### CODE REVIEW

File: OH-Archive/Gemfile [hunk 1]
```diff
@@ -27,3 +27,5 @@ end
 
 # Performance-booster for watching directories on Windows
 gem "wdm", "~> 0.1.1", :platforms => [:mingw, :x64_mingw, :mswin]
+
+gem 'webrick'
```
Comment (flicksolutions) [inline OH-Archive/Gemfile:31]: why did you add that? Was this necessary to run on your local system?

Comment (flicksolutions) [reply OH-Archive/Gemfile:31]: ah I see, I think with newer ruby versions this is needed...
I'm going to run an update on the whole bundle and the npm packages afterwards. so this can stay.

File: OH-Archive/_layouts/default.html
Comment (flicksolutions) [inline OH-Archive/_layouts/default.html:1]: We do not need a button-Element and also no fancy javascript mouse handlers...

Remove <button>-altogether and just add:
```suggestion
                    <a class="btn btn-outline-dark" style="margin-right: 5px" href="https://www.hist.unibe.ch/ueber_uns/personen/falk_francesca/index_ger.html" target="_blank" style="color: black; text-decoration: none;">
                        Kontakt
                    </a>
```

Comment (flicksolutions) [reply OH-Archive/_layouts/default.html:1]: As you can see, there is no change in the visual. The semantics of the HTML is clear (it's just a link) and the code is simpler.

File: OH-Archive/_layouts/homePage.html
Comment (flicksolutions) [inline OH-Archive/_layouts/homePage.html:1]: same as default

File: OH-Archive/_layouts/post.html [hunk 2]
```diff
@@ -40,10 +40,11 @@
 <div style="padding: 50px;">
         <div class="container newTopbar">
             <div class="d-flex justify-content-start flex-wrap" style="float: right">
-            <button class="btn btn-outline-dark" onclick="window.open('https://www.hist.unibe.ch/ueber_uns/personen/falk_francesca/index_ger.html', '_blank');"
-                    type="button" style="margin-right: 5px">
-                Kontakt
-            </button>
+                <button class="btn btn-outline-dark" style="margin-right: 5px"  onmouseover="document.querySelector('.btn a').style.color='white';" onmouseout="document.querySelector('.btn a').style.color='black';">
```
Comment (flicksolutions) [inline OH-Archive/_layouts/post.html]: again, same as in the default

File: OH-Archive/_layouts/post.html [hunk 3]
```diff
@@ -112,24 +113,12 @@ <h2 style="text-align: center">{{content}}
                 <div class="row" style="padding: 15px">
                     <h1> Weitere Informationen zum Interview</h1>
                     <div class="col">
-                        {% if page.birthday != "ubk" %}
```
Comment (flicksolutions) [inline OH-Archive/_layouts/post.html:115]: good, but as you can see in the .md-files: The data in question is still in there. Please remove id from the posts pages aswell.

To do this: adjust the md-generation here: `OH-Archive/backend/generate-md-files.js`

remove line 72: `                    + "\nlocation: " + geo.hasGeoloc`
and line 71: `                    + "\ncreator: " + general.creator`

File: OH-Archive/_posts/2023-05-03-Marie-SaulnierBloch.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: Interview mit Marie Saulnier Bloch
+collection: 'Einbürgerungen à la «Schweizermacher»?'
+protagonist: Marie Saulnier Bloch
+creator: Amanda Unger
+birthday: 26.05.1974
```
Comment (flicksolutions) [inline OH-Archive/_posts/2023-05-03-Marie-SaulnierBloch.md]: this shouldn't be here after the change in the md-generation-file

File: OH-Archive/_posts/2023-05-03-Marie-SaulnierBloch.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+location: Bern BE (Schweiz)
```
Comment (flicksolutions) [inline OH-Archive/_posts/2023-05-03-Marie-SaulnierBloch.md]: this should also be gone.

File: OH-Archive/assets/js/indexScript.js [hunk 6]
```diff
@@ -122,7 +122,7 @@ request.onload = function () {
                 const h4 = document.createElement("h4");
                 h4.setAttribute("style","text-align: left;");
 
-                const h4Text = document.createTextNode(general.isSubjof);
+                const h4Text = document.createTextNode(general.isSubjof.replace(/(")([^"]*)(")/g, '«$2»'));
```
Comment (flicksolutions) [inline OH-Archive/assets/js/indexScript.js]: I don't like this.
The mistake is in the data. If we change the string here it will lead to confusion. The Data itself has to be changed... I did this in omeka.

Revert this change.

File: OH-Archive/publication.html [hunk 7]
```diff
@@ -14,10 +14,12 @@
 <div id='id-Saisonnières-und-Saisonniers' class="container" style="padding-top: 15px; text-align: left">
     <h3 style="text-align: left;">Kontext zu Saisonnières und Saisonniers</h3>
     <div style="font-size: 20px;">
-        Seminar: «Wir, Saisonniers und Saisonnières…» Für eine Ausstellung im NMB (Neues Museum Biel) Oral History Interviews mit ehemaligen Saisonniers und Saisonnières durchführen und einen Podcast produzieren </div>
-    <div id='id-Saisonnières-und-Saisonniers-links'>
-        <a href="https://www.ksl.unibe.ch/KSL/kurzansicht?9&stammNr=472151&semester=HS2021&lfdNr=0"  target="_blank" class="link-primary" style="font-size: 20px;">Kursbeschreibung</a>
+        Das Neue Museum Biel wird 2022 die Arbeits- und Lebensbedingungen der Saisonniers und Saisonnières beleuchten. in diesem Zusammenhang werden wir in unserem Seminar Oral History Interviews mit Arbeiter*innen führen, die unter dem Saisonnier-Statut in der Region Biel arbeiteten. Anschliessend werden wir auf der Grundlage dieser Interviews Podcasts als Begleitmaterial zur Ausstellung erstellen.
```
Comment (flicksolutions) [inline OH-Archive/publication.html]: add wrapping paragraphs to make the text more legible.
```suggestion
        <p>Das Neue Museum Biel wird 2022 die Arbeits- und Lebensbedingungen der Saisonniers und Saisonnières beleuchten. in diesem Zusammenhang werden wir in unserem Seminar Oral History Interviews mit Arbeiter*innen führen, die unter dem Saisonnier-Statut in der Region Biel arbeiteten. Anschliessend werden wir auf der Grundlage dieser Interviews Podcasts als Begleitmaterial zur Ausstellung erstellen.</p>
```

File: OH-Archive/publication.html [hunk 8]
```diff
@@ -14,10 +14,12 @@
...
+Oral History ist eine wichtige Methode der Zeitgeschichte, um Sachverhalte zu erhellen, die wenig oder schlecht dokumentiert sind. Sie zeichnet Erinnerungen von Zeitzeug*innen auf und wertet diese aus. Dadurch wird es möglich, die Perspektiven von Menschen sichtbar zu machen, die in der bisherigen Geschichtsschreibung oft zu wenig berücksichtigt wurden.
```
Comment (flicksolutions) [inline OH-Archive/publication.html]:
```suggestion
<p>Oral History ist eine wichtige Methode der Zeitgeschichte, um Sachverhalte zu erhellen, die wenig oder schlecht dokumentiert sind. Sie zeichnet Erinnerungen von Zeitzeug*innen auf und wertet diese aus. Dadurch wird es möglich, die Perspektiven von Menschen sichtbar zu machen, die in der bisherigen Geschichtsschreibung oft zu wenig berücksichtigt wurden.</p>
```

File: OH-Archive/publication.html [hunk 9]
```diff
@@ -14,10 +14,12 @@
...
+Das Saisonnier-Statut war in der Schweiz ab den 1930er Jahren bis 2002 in Kraft. Temporäre Arbeitskräfte sollten während einer Saison (meist März bis Ende November) in der Schweiz arbeiten. Insbesondere in den Zeiten des Wirtschaftsbooms machte die Schweiz ausgiebig von dieser Bewilligung Gebrauch. Auf diese Weise sollten flexibel einsetzbare und zugleich möglichst «rechtlose» Arbeitskräfte für die Industrie, die Bau- und Landwirtschaft und den Tourismus gewonnen werden. Nach meist neun Monaten mussten diese Migrant*innen wieder ausreisen, oft ohne zu wissen, ob sie im kommenden Jahr wieder angestellt würden. Mit der so genannten A-Bewilligung waren zudem weitere Einschränkungen verbunden. Die Saisonniers und Saisonnières konnten während der laufenden Saison weder die Arbeitsstelle wechseln noch ihren Wohnsitz in einen anderen Kanton verlegen. Auch um ihre soziale Sicherheit war es schlecht gestellt, denn lange hatten sie etwa auch keinen Anspruch auf Arbeitslosengeld. Zudem war der Familiennachzug für Ehepartner und Kinder verboten.
```
Comment (flicksolutions) [inline OH-Archive/publication.html]:
```suggestion
<p>Das Saisonnier-Statut war in der Schweiz ab den 1930er Jahren bis 2002 in Kraft. Temporäre Arbeitskräfte sollten während einer Saison (meist März bis Ende November) in der Schweiz arbeiten. Insbesondere in den Zeiten des Wirtschaftsbooms machte die Schweiz ausgiebig von dieser Bewilligung Gebrauch. Auf diese Weise sollten flexibel einsetzbare und zugleich möglichst «rechtlose» Arbeitskräfte für die Industrie, die Bau- und Landwirtschaft und den Tourismus gewonnen werden. Nach meist neun Monaten mussten diese Migrant*innen wieder ausreisen, oft ohne zu wissen, ob sie im kommenden Jahr wieder angestellt würden. Mit der so genannten A-Bewilligung waren zudem weitere Einschränkungen verbunden. Die Saisonniers und Saisonnières konnten während der laufenden Saison weder die Arbeitsstelle wechseln noch ihren Wohnsitz in einen anderen Kanton verlegen. Auch um ihre soziale Sicherheit war es schlecht gestellt, denn lange hatten sie etwa auch keinen Anspruch auf Arbeitslosengeld. Zudem war der Familiennachzug für Ehepartner und Kinder verboten.</p>
```

File: OH-Archive/publication.html [hunk 10]
```diff
@@ -44,9 +47,9 @@ <h3 style="text-align: left;">Publikationen zu tbd-wird ergänzt</h3>
 
 <div id="id-Einbürgerungen-à-la-«Schweizermacher»?" class="container" style="padding-top: 15px; text-align: left">
     <h3 style="text-align: left;"> Kontext zu «Einbürgerungen à la Schweizermacher»</h3>
-    <div style="font-size: 20px;"> Seminar: Einbürgerungen à la „Schweizermacher“</div>
-    <div id="id-Einbürgerungen-à-la-«Schweizermacher»?-links">
-        <a href="https://www.ksl.unibe.ch/KSL/kurzansicht?0&stammNr=481620&semester=FS2023&lfdNr=0" target="_blank" class="link-primary" style="font-size: 20px;">Kursbeschreibung</a><br></div>
+    <div style="font-size: 20px;"> 1978 kam der Film „Die Schweizermacher“, der die schweizerische Einbürgerungspraxis aufs Korn nahm, ins Kino. Diese ironisch-satirische Komödie handelte unter anderem von der Tätigkeit zweier Kantonspolizisten, deren Aufgabe darin bestand, einbürgerungswillige Ausländer:innen zu überprüfen. Die Vorkommnisse, die im Spielfilm in Szene gesetzt werden, waren fiktional. Doch in der Beschreibung dieses äusserst erfolgreichen Films heisst es: „Alles zwar frei erfunden, aber Ähnlichkeiten mit tatsächlichen Verhältnissen sind nicht ganz ausgeschlossen ...“.
```
Comment (flicksolutions) [inline OH-Archive/publication.html]: Same here. add wrapping <p>

File: OH-Archive/publication.html [hunk 11]
```diff
@@ -30,10 +32,11 @@ <h3 style="text-align: left;">Kontext zur «Schwarzenbach-Abstimmung»</h3>
 <div id="id-Wenn-Migration-die-Arbeitsteilung-in-der-Familie-prägt" class="container" style="padding-top: 15px; text-align: left">
     <h3 style="text-align: left;">Kontext zu «Über Frauen als Ernährerinnen und Hausmänner: Wenn Migration die Arbeitsteilung in der Familie prägt»</h3>
     <div style="font-size: 20px;">
-        Seminar: «Über Frauen als Ernährerinnen und Hausmänner»  </div>
+        Bei den Migrantinnen, die in den wirtschaftlichen Boomjahren der Nachkriegszeit in die Schweiz kamen, waren oft Vollzeit erwerbstätig, auch wenn sie Kinder hatten. Mit ihren Partnern praktizierten sie dabei zuweilen gezwungenermassen andere Formen der Verteilung von Hausarbeit und Kinderbetreuung als das damalige Familienmodell mit dem Mann als Ernährer und der Frau als Mutter und Hausfrau vorsah. Hatten die Männer schlechtere Chancen auf dem Arbeitsmarkt – was etwa bei binationalen Ehen nicht selten der Fall war – übernahmen diese manchmal auch die Hauptlast der Hausarbeit und Kinderbetreuung.
```
Comment (flicksolutions) [inline OH-Archive/publication.html]: add wrapping <p>

**Review by flicksolutions — CHANGES_REQUESTED** [review]

File: OH-Archive/assets/js/indexScript.js
Comment (flicksolutions) [inline OH-Archive/assets/js/indexScript.js:1]: @ZEDz318 
because I changed the title with the quotation marks in the DB, please change `"Schwarzenbach-Abstimmung"` on line 134 to `«Schwarzenbach-Abstimmung»`

File: OH-Archive/_layouts/default.html
Comment (ZEDz318) [reply OH-Archive/_layouts/default.html:1]: This makes a lot of sense.... with an anchor inside a button i had to handle the case where the mouse hovers over the button but not the anchor (where both the button and text become black...)

File: OH-Archive/_layouts/homePage.html
Comment (ZEDz318) [reply OH-Archive/_layouts/homePage.html:1]: Done

File: OH-Archive/_layouts/post.html (hunk 2 above)
Comment (ZEDz318) [reply OH-Archive/_layouts/post.html]: Done

File: OH-Archive/_layouts/post.html (hunk 3 above)
Comment (ZEDz318) [reply OH-Archive/_layouts/post.html:115]: You mean line 72 and 73? : 

```
 + "\nbirthday: " + person.birthday.toString().replace(/-/g, ".")
                    + "\nlocation: " + geo.hasGeoloc
```

since general.creator is used in line 125 in post.html : 

` <h2>Aufgenommen am {{page.interviewDate}} von {{page.creator}}</h2>`

I'll wait for you to confirm this before doing any changes

**Comment by flicksolutions** [comment]
Confirmed, yes.
Sorry for the line mixup

ZEDz318 ***@***.***> schrieb am Do., 2. Nov. 2023, 23:13:

File: OH-Archive/_posts/2023-05-03-Marie-SaulnierBloch.md (hunk 4 above)
Comment (ZEDz318) [reply OH-Archive/_posts/2023-05-03-Marie-SaulnierBloch.md]: same for above, I deleted all except creator

File: OH-Archive/_layouts/post.html (hunk 3 above)
Comment (ZEDz318) [reply OH-Archive/_layouts/post.html:115]: also another thing, since we are no longer in need of these elements, should we also remove them from the "Entry" class in indexScript.js?

File: OH-Archive/publication.html (hunk 7 above)
Comment (ZEDz318) [reply OH-Archive/publication.html]: Done

File: OH-Archive/publication.html (hunk 8 above)
Comment (ZEDz318) [reply OH-Archive/publication.html]: Done

File: OH-Archive/publication.html (hunk 9 above)
Comment (ZEDz318) [reply OH-Archive/publication.html]: Done

File: OH-Archive/publication.html (hunk 10 above)
Comment (ZEDz318) [reply OH-Archive/publication.html]: Done

File: OH-Archive/publication.html (hunk 11 above)
Comment (ZEDz318) [reply OH-Archive/publication.html]: Done

File: OH-Archive/assets/js/indexScript.js
Comment (ZEDz318) [reply OH-Archive/assets/js/indexScript.js:1]: Done

File: OH-Archive/_layouts/post.html (hunk 3 above)
Comment (flicksolutions) [reply OH-Archive/_layouts/post.html:115]: yers, this is a very good remark! I'll do that!

**Review by flicksolutions — APPROVED** [review]
